src/list.scss

```diff
@@ -101,6 +101,107 @@ $block: #{$fd-namespace}-list;
         background: var(--sapList_Background);
       }
     }
+
+    &--action {
+      display: -webkit-box;
+      display: flex;
+      size: 2.5rem;
+      padding: 0 0.25rem;
```

Comment:
The specs say that Action List Item inherits from List so I don't this you should set these values here.

Reply:
Right now list does not have these values, so I added in the action list item. I will remove it once the list is updated with the values.